platform/ios/src/MGLMapView.h

```diff
@@ -212,7 +212,7 @@ IB_DESIGNABLE
 /** 
  Currently active style classes, represented as an array of string identifiers.
```

Comment:
Also, because jazzy doesn’t currently display deprecation messages, we usually remove the documentation comment from deprecated symbols.